plugins/org.wso2.developerstudio.eclipse.esb.project/src/org/wso2/developerstudio/eclipse/esb/project/ui/wizard/CloudConnectorImportWizard.java

```diff
@@ -48,224 +48,227 @@
+         */
+        storeWizardPage.getSelectedProject().refreshLocal(IResource.DEPTH_INFINITE, null);
+    }
 
-	private boolean downloadConnectorAndUpdateProjects(String downloadLink) {
-		String zipDestination = null;
-		try {
-			URL url = new URL(downloadLink);
-			String[] segments = downloadLink.split("/");
-			String zipFileName = segments[segments.length - 1];
-			String parentDirectoryPath = storeWizardPage.getSelectedProject().getWorkspace().getRoot().getLocation()
-					.toOSString()
-					+ File.separator + DIR_DOT_METADATA + File.separator + DIR_CONNECTORS;
-			File parentDirectory = new File(parentDirectoryPath);
-			if (!parentDirectory.exists()) {
-				parentDirectory.mkdir();
-			}
-			zipDestination = parentDirectoryPath + File.separator + zipFileName;
-			InputStream is = url.openStream();
-			File targetFile = new File(zipDestination);
-			targetFile.createNewFile();
-			FileOutputStream outputStream = new FileOutputStream(targetFile);
-			int bytesRead = -1;
-			byte[] buffer = new byte[BUFFER_SIZE];
-			while ((bytesRead = is.read(buffer)) != -1) {
-				outputStream.write(buffer, 0, bytesRead);
-			}
-			outputStream.close();
-			is.close();
-			updateProjects(zipDestination);
-			return true;
-		} catch (ZipException e) {
-			log.error("Error while extracting the connector zip : " + zipDestination, e);
-		} catch (CoreException e) {
-			log.error("Cannot refresh the project", e);
-		} catch (MalformedURLException malformedURLException) {
-			log.error("Malformed connector URL provided : " + downloadLink, malformedURLException);
-		} catch (IOException e) {
-			log.error("Error while downloading connector : " + downloadLink, e);
-		}
-		return false;
-	}
+    private boolean downloadConnectorAndUpdateProjects(String downloadLink) {
+        String zipDestination = null;
+        try {
+            URL url = new URL(downloadLink);
+            String[] segments = downloadLink.split("/");
+            String zipFileName = segments[segments.length - 1];
+            String parentDirectoryPath = storeWizardPage.getSelectedProject().getWorkspace().getRoot().getLocation()
+                    .toOSString() + File.separator + DIR_DOT_METADATA + File.separator + DIR_CONNECTORS;
+            File parentDirectory = new File(parentDirectoryPath);
+            if (!parentDirectory.exists()) {
+                parentDirectory.mkdir();
+            }
+            zipDestination = parentDirectoryPath + File.separator + zipFileName;
+            InputStream is = url.openStream();
+            File targetFile = new File(zipDestination);
+            targetFile.createNewFile();
+            FileOutputStream outputStream = new FileOutputStream(targetFile);
+            int bytesRead = -1;
+            byte[] buffer = new byte[BUFFER_SIZE];
+            while ((bytesRead = is.read(buffer)) != -1) {
+                outputStream.write(buffer, 0, bytesRead);
+            }
+            outputStream.close();
+            is.close();
```

Comment:
These, IO streams should be closed within finally block.

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/832